src/AdminCommands.ts

```diff
@@ -304,9 +331,12 @@ export class AdminCommands {
         return new AdminCommand(
             "help [command]",
             "describes the commands available",
-            ({respond, command}) => {
+            ({respond, command}: {
+                respond: ResponseCallback,
+                command?: string,
+            }) => {
                 if (command) {
-                    const cmd = this[command as string] as AdminCommand;
+                    const cmd = this.commands.find((adminCommand) => (adminCommand.command.split(' ')[0] === command));
```

Comment:
This is the line that fixes the `help oauth` and protects against calling `.detailedDescription()` on other class members.